gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -280,11 +280,11 @@ private static void setupObjectsProject(Project project, Supplier<GeneratorOptio
         project.getDependencies().add("api", Dependencies.JETBRAINS_ANNOTATIONS);
     }
 
-    private static void setupDialogueProject(Project project, Supplier<GeneratorOptions> _optionsSupplier) {
+    static void setupDialogueProject(Project project, Supplier<GeneratorOptions> _optionsSupplier) {
```

Comment:
Followed the pattern of `ignoreFromCheckUnusedDependencies` from below (which is also made package-private and [reused in ConjureJavaLocalCodegenPlugin](https://github.com/palantir/gradle-conjure/blob/4c21b4d91ee7d6d39884f531ffd5688d39e47bf6/gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPlugin.java#L142), same as a couple of other methods), though I believe this all should probably be extracted to a common class